.openpublishing.redirection.json

```diff
@@ -324,6 +324,21 @@
             "source_path": "entity-framework/core/modeling/relational/index.md",
             "redirect_url": "/ef/core/modeling/",
             "redirect_document_id": false
+        },
+        {
+            "source_path": "entity-framework/core/what-is-new/ef-core-3.0/index.md",
+            "redirect_url": "/ef/core/what-is-new/ef-core-3.x",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md",
+            "redirect_url": "/ef/core/what-is-new/ef-core-3.x/breaking-changes",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "entity-framework/core/miscellaneous/1x-2x-upgrade.md",
+            "redirect_url": "/ef/core/what-is-new/ef-core-2.0/upgrade",
+            "redirect_document_id": false
```

Comment:
add redirection for all deleted/renamed content.

Reply:
Apart from the two deleted pages (1.0 rc1->rc2, rc2->RTM) have I missed anything?

Reply:
/core/what-is-new/efc-core-2.0.md file moved.
It should update its uid and add redirect for it here.
Couldn't comment on just renamed file. Git UI. :trollface:

Reply:
Unless I'm mistaken, that page hasn't actually changed its URL, because it has become the index in a directory of the same name... I tried to define that redirect and got a circular redirect loop warning..